### ISSUE TITLE
`krakenrs::ws::BookData` doesn't implement `std::fmt::Debug`

### ISSUE THREAD

**Issue by gitmalong** [cbeck88/krakenrs#6]
Tried the websockets code from the readme

```rust
    use krakenrs::{KrakenWsConfig, KrakenWsAPI};
    use std::{
        collections::BTreeMap,
        time::Duration,
        thread,
    };

    fn main() {
        let pairs = vec!["XBT/USD".to_string(), "SOL/XBT".to_string()];

        let ws_config = KrakenWsConfig {
            subscribe_book: pairs.clone(),
            book_depth: 10,
        };
        let api = KrakenWsAPI::new(ws_config).expect("could not connect to websockets api");

        loop {
            thread::sleep(Duration::from_millis(500));
            let books = pairs
                .iter()
                .map(|pair| (pair.clone(), api.get_book(pair)))
                .collect::<BTreeMap<_, _>>();

            for (pair, book) in books {
                println!("{}", pair);
                println!("{:?}", book); // error occurs in here
            }
            if api.stream_closed() { return; }
        }
    }
```

**Comment by cbeck88** [comment]
thanks, will fix

**Comment by cbeck88** [comment]
I think this is fixed in #7
